src/helpers/render-document-file.js

```diff
@@ -178,6 +178,7 @@ function findXMLEquivalent(docxDocumentInstance, vNode, xmlFragment) {
           lineHeight: Math.max(lineHeight, 240),
           strong: 'bold',
           beforeSpacing: 240,
+          heading: `Heading${vNode.tagName[1]}`,
```

Comment:
this should be renamed as paragraphStyle

Reply:
Oops, fixed it